### PR TITLE
Add custom PR comment url

### PR DESCRIPTION
**Pull request by alexjurkiewicz**
Allows commenting on PRs / issues from outside `pull_request` workflows.

### CODE REVIEW

**Comment by alexjurkiewicz** [comment]
I've updated this to remove merge conflicts.

File: examples/comments-non-pull-request.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        with:
+          tf_actions_version: latest
+          tf_actions_subcommand: init
+      - name: Terraform Apply
+        uses: hashicorp/terraform-github-actions@master
+        with:
+          tf_actions_version: latest
+          tf_actions_subcommand: apply
+          tf_actions_comment: true # default, but being explicit
+          tf_actions_comment: ${{ steps.load-pr.outputs.comments_url }}
```
Comment (dnorth98) [inline examples/comments-non-pull-request.md]: I think this should be `tf_actions_comment_url`

**Comment by dnorth98** [comment]
This PR is brilliant.  I've just ran into this exact problem while trying to run the apply phase from a PR comment (similar to the workflow for [atlantis](https://www.runatlantis.io/).

**Comment by alexjurkiewicz** [comment]
Wow. Atlantis looks great. I hadn't seen it. I wonder how hard it would be
to adapt to running in GitHub Action's...

File: examples/comments-non-pull-request.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        id: load-pr
+        run: |
+          set -eu
+          resp=$(curl -sSf \
+            --url ${{ github.event.issue.pull_request.url }} \
+            --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+            --header 'content-type: application/json')
+          sha=$(jq -r '.head.sha' <<< "$resp")
+          echo "::set-output name=head_sha::$sha"
+          comments_url=$(jq -r '.comments_url')
```
Comment (dnorth98) [inline examples/comments-non-pull-request.md]: Looks like you're missing the input redirect here as the var is never being set.  Should be

```comments_url=$(jq -r '.comments_url' <<< "$resp")```

**Comment by alexjurkiewicz** [comment]
Thanks for the review. I wrote up the example from a private repo, so it wasn't that well tested!

**Comment by alexjurkiewicz** [comment]
Hey @sudomateo , not sure how disrupted your working life is right now (mine is very!) -- any idea when you can look at this PR?

**Comment by sudomateo** [comment]
@alexjurkiewicz Very disrupted here as well. I'll have to delay reviewing this for at least one week. Sorry about that!

**Comment by sudomateo** [comment]
This `hashicorp/terraform-github-actions` repository is no longer actively developed or maintained. It has been superseded by the [hashicorp/setup-terraform](https://github.com/hashicorp/setup-terraform) GitHub action.

See #174 for more information.

**Pull request closed without merging**